src/components/interfaces.ts

```diff
@@ -23,26 +23,28 @@
  SOFTWARE.
 
  */
-
 import { ILookmlModelExploreField } from "@looker/sdk"
 
 export interface ColumnDescriptor {
   name: string
   label: string
   rowValueDescriptor: string
-  formatter: (
-    x: string,
-    isRow?: boolean,
-    field?: ILookmlModelExploreField,
-    commentCount?: number,
-    canComment?: boolean,
-    reader?: boolean
-  ) => string | JSX.Element
+  formatter: (props: FormatterProps) => JSX.Element
```

Comment:
Is this acceptable or what you would like see changed to `Formatter`?

Reply:
it can be lowercase as a prop, but it needs to be uppercase to be used in a tag

Reply:
also, should be `React.ComponentType<FormatterProps>`. This allows class components to be passed as well.

Reply:
Is it OK to leave class components out of the requirement?

Reply:
If that's your intention, then the type should be `React.FunctionComponent<FormatterProps>`